deploy/redeploy_bot_guard.sh

```
@@ -0,0 +1,59 @@
+echo "BOT_TOKEN: $BOT_TOKEN"
+echo "LOGGING: $LOGGING"
+
+
+# Updating the Git repo
+echo "Updating Git repo..."
+#cd /path/to/your/git/repo
+git pull
+
+# Moving to the deployment directory
+echo "Moving to deployment directory..."
+cd alerts
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)